### PR TITLE
[13.0] [MIG] base_iso3166

### PR DESCRIPTION
**Pull request by ThomasBinsfeld**

### CODE REVIEW

**Review by chrisdec — APPROVED** [review]
functional test

**Review by ernestotejeda — COMMENTED** [review]
Hi @ThomasBinsfeld , you haven't put the pre-commit changes (black, prettier, etc) and v13 migration changes into separate commits.
See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch for more information

File: base_iso3166/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# Copyright 2017  Creu Blanca <www.creublanca.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "ISO 3166",
+    "version": "13.0.1.0.0",
+    "author": "Tecnativa, " "Creu Blanca, " "Odoo Community Association (OCA)",
```
Comment (ernestotejeda) [inline base_iso3166/__manifest__.py:8]:
```suggestion
    "author": "Tecnativa, Creu Blanca, Odoo Community Association (OCA)",
```

File: base_iso3166/models/res_country.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                    c = getattr(pycountry, country_type).get(alpha_2=country.code)
+                except KeyError:
+                    try:
+                        c = getattr(pycountry, country_type).get(alpha2=country.code)
+                    except KeyError:
+                        pass
+                if c:
+                    break
+            if c:
+                country.code_alpha3 = getattr(c, "alpha_3", getattr(c, "alpha3", False))
```
Comment (ernestotejeda) [inline base_iso3166/models/res_country.py:46]:
```suggestion
                country.code_alpha3 = getattr(c, "alpha_3", False)
```
Not the same as this?

**Comment by ernestotejeda** [comment]
Hi @ThomasBinsfeld, if you can't continue with this migration, I can do it in a new PR. What you think? :)

**Comment by pedrobaeza** [comment]
As this doesn't split pre-commit changes from migration ones, we have superseded in #95 if you don't mind, so closing.

**Pull request closed without merging**